themes/default/content/blog/zephyr-code-placement-and-stacks/index.md

```diff
@@ -0,0 +1,140 @@
+
+* Who is responsible for maintaining the code? If the application code and the Pulumi code are managed by different teams, then using separate repositories may be the best approach.
+* What sort of access controls are needed? If the organization needs different access controls for the application code and the Pulumi code, then separate repositories are generally needed.
+* What is the relationship between the infrastructure resources managed by Pulumi and the applications? If the infrastructure is "shared," meaning it is used by multiple applications, then keeping the Pulumi code in a separate repository may be preferred. If the infrastructure is dedicated to a specific application, then keeping the application code and the Pulumi code together may be preferred.
+* What about CI/CD? If an organization will use CI/CD, that may affect how you organize your code. (This is something we'll delve into more deeply later in this series.)
+
+The answers to these questions may change over time as organizations grow and their applications evolve. Addressing that change and its effects on an organization's Pulumi projects and stacks is also something we'll be discussing later in this series.
+
+{{% notes type="info" %}}
+The discussion above is working from the assumption that Pulumi users will store their Pulumi code in a version control system, like Git. Strictly speaking, using a version control system isn't required for Pulumi to function (there is one exception we'll touch on later in this series when we discuss Pulumi Deployments), but we **very strongly** recommend that all users use Pulumi in conjunction with version control.
+{{% /notes %}}
+
+In the case of Zephyr, our fictional company and use case, it is a relatively small organization with a single team of developers that are responsible for  managing both the cloud infrastructure as well as the development of the online store application. For their use case, Zephyr felt like a monorepo approach was most appropriate for their specific requirements. They're using a single Pulumi project---for now. (Stay tuned to this blog series to see how that evolves, and why!)
+
+### Addressing the need for multiple environments
+
+The other issue Zephyr encountered is how to handle the need for multiple instances of their application. In addition to a production instance---which is the instance behind Zephyr's public-facing online store---Zephyr also felt they needed an environment for the developers to use in testing changes to the online store.
+
+This use case---needing to have multiple, separate instances of the infrastructure and applications created by a single Pulumi program---is exactly what Pulumi stacks were designed to address. Each stack is a separate instance of the resources created by a Pulumi program within a project. Further, each stack has its own independent state, and each stack has its own configuration values.
```

Comment:
should we link to the stacks documentation in here somewhere? 
https://www.pulumi.com/docs/intro/concepts/stack/

Reply:
Done in 35635a95